esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -260,6 +259,68 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
     }
     if(m_challenge_realm.length() == 0)
         m_challenge_realm.append("ESP");
+
+    if(!m_secmgr.get())
+        return;
+
+    processName.set(procname);
+    const char* authDomain = bnd_cfg->queryProp("@authDomain");
+    if (!isEmptyString(authDomain))
+        domainName.set(authDomain);
+    else
+        domainName.set("default");
+    domainSessionSDSPath.setf("%s/%s[@name=\"%s\"]/%s[@name=\"%s\"]/", PathSessionRoot, PathSessionProcess,
+        procname, PathSessionDomain, domainName.str());
```

Comment:
Am I right in thinking procname here is shared across all instances of the same ESP?  So all instances of that ESP will share the same sessions?  (As they should because they may be behind a load balancer).
But also bindings on different ports will share the same sessions, unless authDomain is changed?  Might not be good having one port affecting another ports session state.